sdk/javascript/test/symbol/resources/voting_keys_generator_test_vectors.yaml

```diff
@@ -0,0 +1,20 @@
+- name: test_vector_1
```

Comment:
right now this file is duplicated between JS + PY sdks because it is used by unit tests. not sure what to do with it? we could move to testvectors and symlink perhaps?

@gimre-xymcity ?

Reply:
hmm, looking at it, I think we should just move that part into testvectors, and run it as a part of all.js/all.py
(read: it doesn't really feel like a unit test, does it?)

not :100: sure, what do you think?